doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -594,6 +594,159 @@ must be sent.
 *default location*, then the query will be declared as ambigous and an HTTP error response with code ``409``
 must be sent. 
 
+## Notifications formats
+
+Notifications include two fields: `subscriptionId` (which value is the one of the subscription triggering
+the notification) and `data` (which holds the actual data corresponding to entities and attributes).
+
+The `data` field can use different representation depending on two properties (especified as part of the
+corresponding subscription, see subscriptions operations section for details).
+
+* attrsFormat ("full", "keyValues", "values"; default is "full") to specify how the attributes are
```

Comment:
I don't like too much attrsFormat and dataFormat, but is the best names I have found... Please, go ahead if you have any other suggestion.